source/PlayState.hx

```diff
@@ -1,18 +1,83 @@
 package;
 
+import flixel.FlxG;
 import flixel.FlxState;
+import flixel.group.FlxGroup;
+import flixel.text.FlxText;
+import flixel.util.FlxColor;
 
 class PlayState extends FlxState
 {
+	var screensaverGroup = new FlxTypedGroup<ScreensaverHero>();
+	var wallGroup = new FlxTypedGroup<Wall>();
+	var hero:ControllableHero;
+	var wall1:Wall;
+	var wall2:Wall;
+	var wall3:Wall;
+	var wall4:Wall;
+	var screensaver:ScreensaverHero;
+	var screensaver1:ScreensaverHero;
+	var screensaver2:ScreensaverHero;
+	var screensaver3:ScreensaverHero;
+	var screensaver4:ScreensaverHero;
+	var score:Int = 0;
+	var scoreText:FlxText;
+
 	override public function create()
 	{
 		super.create();
-		var ourHero:Hero = new Hero(300, 300, "John");
-		add(ourHero);
+		trace(FlxG.height);
+		trace(FlxG.width);
+
+		var helloWorldText = new FlxText(200, 200, -1);
+		helloWorldText.text = "Hello World!";
+		helloWorldText.setFormat("assets/fonts/AkayaTelivigala-Regular.ttf", 60, FlxColor.BLUE, "center", FlxTextBorderStyle.SHADOW);
+		add(helloWorldText);
+
+		scoreText = new FlxText(10, 10);
+		scoreText.text = "Score: " + Std.string(score);
+		add(scoreText);
+
+		// Added the screen saver stuff
+		screensaver = new ScreensaverHero(10, 200);
+		screensaver1 = new ScreensaverHero(10, 200);
+		screensaver2 = new ScreensaverHero(10, 200);
+		screensaver3 = new ScreensaverHero(10, 200);
+		screensaver4 = new ScreensaverHero(10, 200);
+		screensaverGroup.add(screensaver);
+		screensaverGroup.add(screensaver1);
+		screensaverGroup.add(screensaver2);
+		screensaverGroup.add(screensaver3);
+		screensaverGroup.add(screensaver4);
```

Comment:
Since you don't use these screensaver variables elsewhere anymore, you could simplify this into some kind of loop:
```
for (int i in 0...5) {
    screensaverGroup.add(new Screensaverhero(10, 200);
}
```
similar idea applies to your wall instantiation below, though you have to do a bit of math inside the loop to figure out what their positions should be.

Alternatively, if you do end up needing a sizeable block of code to initialize something more complicated, it could be worthwhile to just make a private method named `initializeScreensaverGroup()` or something similar that creates and sets up and calls `add()` for your screensaver group. Repeat that idea for other things in your state, and your `create` code can be clean(er) and pretty readable, something like

```
override public function create()
{
    initializeHelloWorld();
    initalizeScoreText();
    initializeScreensaverGroup();
    initializeHero();
    initializeWalls();
}
```
This sort of thing can be a real lifesaver if you're collaborating with others in a project, since it tends to minimize the complexity of conflict in methods like `create()` that lots of people might be interested in changing.

No change necessary, but good to consider.